src/create.html

```diff
@@ -17,7 +17,31 @@
     <header class="app-header">
       <h1 class="app-header__heading">Create</h1>
     </header>
-    <main class="app-main"></main>
+    <main class="app-main">
+      <section class="create-form">
```

Comment:
You don't necessarily need this sectioning element. The `form` itself is a sectioning element.

Reply:
Changed it.